web-local/src/lib/components/workspace/core/WorkspaceHeaderStatusSpinner.svelte

```diff
@@ -27,10 +27,10 @@
   $: applicationStatusTooltip = applicationStatusTooltipMap[applicationStatus];
 </script>
 
-<div>
+<div class="mr-2">
   <div class="text-gray-400">
     <Tooltip alignment="center" distance={16} location="left">
```

Comment:
```suggestion
    <Tooltip alignment="center" distance={8} location="left">
```

16 feels a little too far from the icon given the other header spacing